osidb/tests/test_flaw.py

```diff
@@ -150,6 +150,37 @@ def test_create(self, datetime_with_tz, good_cve_id):
         # assert vuln_1.delete()
         # assert vuln_2.delete()
 
+    def test_create_flaw_method(self):
+        acls = [
+            uuid.uuid5(
+                uuid.NAMESPACE_URL,
+                "https://osidb.prod.redhat.com/ns/acls#data-prodsec",
+            )
+        ]
+        Flaw.objects.all().delete()
+        Flaw.objects.create_flaw(
+            bz_id="12345",
```

Comment:
If we pass it `bz_id` and it doesn't exist, `create_flaw` should pass the `bz_id` value to `meta_attr` so that we don't have to explicitly pass both `bz_id` and `meta_attr__bz_id`

Reply:
That makes sense. I will extend the `create_flaw` method with it.